<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigrationWindow</class>
 <widget class="QMainWindow" name="ConfigrationWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigrationWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhLowercaseOnly</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(218, 218, 218);
background-color: rgb(52, 52, 52);
alternate-background-color: rgb(83, 83, 83);
border-color: rgb(113, 113, 113);
selection-background-color: rgb(131, 131, 131);
selection-color: rgb(222, 222, 222);
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CompassGames, All rights reserved 2012</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="Graphics_view_gridLayout"/>
    </item>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	color: rgb(200, 200, 200);
	border-color: rgb(173, 173, 173);
	background-color: rgb(83, 83, 83);
}

QPushButton:disabled
{
color: rgb(90, 90, 90);
background-color: rgb(66, 66, 66);
border:0px
}

QComboBox {
	background-color: rgb(83, 83, 83);
}

QLineEdit {
	background-color: rgb(83, 83, 83);
	border-color: rgb(77, 77, 77);
}

QLineEdit:disabled {
	background-color: rgb(180,180,180);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>596</width>
         <height>695</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="12" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="refresh">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="slowAnimationPushButton">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Slow Motion</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="MapInfo">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>199</width>
             <height>171</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Map Info</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="2">
             <widget class="QLineEdit" name="MapXMLNamelineEdit">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>XML: </string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Name: </string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="MapNamelineEdit"/>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="MapBackgroundlineEdit"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Background :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QLabel" name="label_11">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Map Info</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QPushButton" name="loadXMLpushButton">
              <property name="text">
               <string>Load</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QCheckBox" name="MapCheckPointcheckBox">
              <property name="text">
               <string>Check Point</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>MapXMLNamelineEdit</zorder>
           <zorder>label_3</zorder>
           <zorder>label_4</zorder>
           <zorder>MapNamelineEdit</zorder>
           <zorder>MapBackgroundlineEdit</zorder>
           <zorder>label_6</zorder>
           <zorder>label_11</zorder>
           <zorder>loadXMLpushButton</zorder>
           <zorder>verticalSpacer</zorder>
           <zorder>MapCheckPointcheckBox</zorder>
          </widget>
          <widget class="QWidget" name="Elementproperties">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>578</width>
             <height>557</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Element properties</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0" colspan="5">
             <widget class="QListWidget" name="ElementlistWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="tabKeyNavigation">
               <bool>false</bool>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Name: </string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="ElementNamelineEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Z: </string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="ElementZlineEdit"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>SVG or XML: </string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="ElementSVGlineEdit"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Type :</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QComboBox" name="ElementTypeComboBox"/>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="AddElementpushButton">
              <property name="text">
               <string>Add Element</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="5">
             <widget class="QLabel" name="label_13">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Element properties</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QPushButton" name="ElementXDragDroppushButton">
              <property name="text">
               <string>X/DD</string>
              </property>
             </widget>
            </item>
            <item row="10" column="2">
             <widget class="QLineEdit" name="ElementXlineEdit"/>
            </item>
            <item row="11" column="0">
             <widget class="QPushButton" name="ElementYDragDroppushButton">
              <property name="maximumSize">
               <size>
                <width>16000000</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Y/DD</string>
              </property>
             </widget>
            </item>
            <item row="11" column="2">
             <widget class="QLineEdit" name="ElementYlineEdit"/>
            </item>
            <item row="12" column="0">
             <widget class="QPushButton" name="ElementRotationDragDroppushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16000000</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>R/DD</string>
              </property>
             </widget>
            </item>
            <item row="12" column="2">
             <widget class="QLineEdit" name="ElementRotationlineEdit"/>
            </item>
            <item row="20" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Go to :</string>
              </property>
             </widget>
            </item>
            <item row="20" column="2">
             <widget class="QLineEdit" name="ElementXMLlineEdit"/>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="DeleteElementPushButton">
              <property name="text">
               <string>DeleteMe</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0" colspan="3">
             <widget class="QCheckBox" name="ElementFlippedcheckBox">
              <property name="text">
               <string>Flipped</string>
              </property>
             </widget>
            </item>
            <item row="16" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Reward type:</string>
              </property>
             </widget>
            </item>
            <item row="16" column="2">
             <widget class="QComboBox" name="ElementRewardTypeComboBox">
              <item>
               <property name="text">
                <string>money</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>item</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>weapon</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>key</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Quest Key (.svg):</string>
              </property>
             </widget>
            </item>
            <item row="14" column="2">
             <widget class="QLineEdit" name="ElementKeylineEdit"/>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Scale:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLineEdit" name="ElementScalelineEdit"/>
            </item>
            <item row="6" column="0" colspan="3">
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Use XML for any Quest or DummySkelton</string>
              </property>
             </widget>
            </item>
            <item row="21" column="0" colspan="3">
             <widget class="QLabel" name="label_12">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Use area XML for gates, use numbers if reward is money, use key svg if reward is key, use weapon xml if reward is weapon, if the reward is item use the item svg</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="9" column="0" colspan="3">
             <widget class="QLabel" name="label_16">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Scale does not work with Quest or DummySkelton</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="EnemyProperties">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>228</width>
             <height>310</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Enemy Properties</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QCheckBox" name="EnemycheckBox">
              <property name="text">
               <string>Enable random battle</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="6">
             <widget class="QLabel" name="Enemylabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Enemy properties</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="EnemyLevellabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Enemy level:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="6">
             <widget class="QListWidget" name="EnemylistWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="EnemyXMLlabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Enemy XML:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="EnemyAddpushButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add enemy</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="EnemyDeletepushButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delete enemy</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLineEdit" name="EnemyLevellineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLineEdit" name="EnemyXMLlineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="ghostLabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>ghost xml :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="ghostXMLlineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
